docs/en_US/ch1/exchangis_deploy_en.md

```diff
@@ -191,7 +214,7 @@ As shown in the figure below:
 
 #### 2.7.1  Get the front-end installation package 
 
-Exchangis has provided compiled front-end installation package by default, which can be downloaded and used directly ：[Click to jump to the Release interface](https://osp-1257653870.cos.ap-guangzhou.myqcloud.com/WeDatasphere/Exchangis/exchangis1.0.0/dist.zip)
+Exchangis has provided compiled front-end installation package by default, which can be downloaded and used directly ：[Click to jump to the Release interface](https://github.com/WeBankFinTech/Exchangis/releases/download/release-1.0.0-rc1/web-dist.zip)
```

Comment:
Must modify it.

Reply:
It has been updated with the latest link.